src/system/UsersModule/Helper/MailHelper.php

```diff
@@ -113,7 +113,7 @@ public function createAndSendRegistrationMail(UserEntity $userEntity, $userNotif
             $notificationEmail = $this->variableApi->get('ZikulaUsersModule', UsersConstant::MODVAR_REGISTRATION_ADMIN_NOTIFICATION_EMAIL, '');
             if (!empty($notificationEmail)) {
                 $authMapping = $this->authenticationMappingRepository->getByZikulaId($userEntity->getUid());
-                $rendererArgs['isVerified'] = $authMapping->isVerifiedEmail();
+                $rendererArgs['isVerified'] = null === $authMapping || $authMapping->isVerifiedEmail();
```

Comment:
I guess my only remaining concern about this is that I was trying to separate the users and ZAuth as much as possible with the idea that ZAuth could even possibly be disabled or removed. That may still be impossible with or without this PR but this ties them together more, not less.